designs/senya/src/index.mjs

```diff
@@ -0,0 +1,15 @@
+import { Design } from '@freesewing/core'
+import { data } from '../data.mjs'
+// Parts
+import { front } from './front.mjs'
+import { back } from './back.mjs'
+import { bottom } from './bottom.mjs'
+
```

Comment:
You'll need to import and re-export the translations as `i18n`

Reply:
Done, although I think I must've missed something, because the translations don't show up in the lab. :/